packages/botonic-react/src/utils.js

```diff
@@ -92,3 +92,37 @@ export function renderComponent({ renderBrowser, renderNode }) {
   else if (isNode()) return renderNode()
   throw new Error('Unexpected process type. Not recognized as browser nor node')
 }
+
+export const isIphone = () => {
+  // Detecting iOS solution, relying on navigator.platform: https://stackoverflow.com/a/9039885
+  return !!navigator.platform && /iPhone/.test(navigator.platform)
+}
+
+const getWebchatElement = () => document.getElementById('botonic-webchat')
+
+const setWebchatElementHeight = newHeight => {
+  getWebchatElement().style.height = newHeight
+}
+
+export const handleIphoneOnFocus = () => {
```

Comment:
we should NEVER have "utils.js". At least it should be called string_utils.js , file_utils.js...
I suggest moving all iphone functions to iphone_utils.js 

still, utils are also not so extensible, I think it's not too hard to have st like device_config.js
```
class DeviceConfig {
 private isIPhone() {
}
 onFocus() {...}
 onBlur() {...}
 fontsize() {...}
}

const deviceConfig = new DeviceConfig()
```
In the future we may have a subclass IphoneConfig, androidConfig ,....
In this way:
* we won't have to modify the usage of the functions, only the implementation
* we can make DeviceConfig grow instead of patching the TextArea
in summary, **think big, but be lazy** :-)

Reply:
I have moved it into a `iphone-utils.hs` at this moment. I agree that we can solve this in lots of different better ways, but first I would like to check the solution is acceptable for our needs. Also, this is a very specific solution for iPhones/Safari browsers. If we can try it with @guinii and works as expected, I can invest some time trying to do it in the way you suggest 💪

Reply:
Approved, thanks for moving it! you can merge it :+1: 

"this is a very specific solution for iPhones/Safari browsers"
This is **precisely** the reason why you cannot mix a specific solution with the rest of the application or mix it with utils. Hacks and tunings should always be separated, because they tend to get more complex. If the 3 iphone utils need to go together, in the future they should be eg. in a HackForIphoneKeyboard class with a link to the post link you added in the PR